### PR TITLE
Controllers spec

### PR DESCRIPTION
**Pull request by braincee**
**_In this controller-specs branch, I did the following requirements:_**

Project requirements
Add RSpec as a gem to my project.
Create a Request spec file for all app controllers.
Made sure that for each action I check:
If the response status was correct.
If a correct template was rendered.
If the response body includes the correct placeholder text.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    # @user = User.find(params[:user_id])
+    # @posts = @user.posts
+  end
+
+  def show
+    # @user = User.find(params[:user_id])
+    # @post = @user.posts.find(params[:id])
+  end
```
Comment (Mihndim2020) [inline app/controllers/posts_controller.rb:10]: - [OPTIONAL] Please kindly remove commented code from your code base as this makes your code base dirty and less readable.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    # @users = User.all
+  end
+
+  def show
+    # @user = User.find(params[:id])
+  end
```
Comment (Mihndim2020) [inline app/controllers/users_controller.rb:8]: - [OPTIONAL] Please kindly remove commented code from your code base as this makes your code base dirty and less readable.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @braincee, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @Mihndim2020 

**_Note: Please I have implemented all the required and optional changes._**

love-thank-you

**Pull request merged**